### PR TITLE
Adding skycraftbr

### PR DESCRIPTION
**Pull request by AimGear**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/skycraftbr/background.png
Comment (DoJapHD) [inline minecraft_servers/skycraftbr/background.png:1]: Compress file

File: minecraft_servers/skycraftbr/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/skycraftbr/background@2x.png:1]: Compress file

File: minecraft_servers/skycraftbr/manifest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "server_wildcards": [
+      "%.skycraft.com.br",
+      "%.skycraft.network",
+      "%.skycraft.games"
+    ],
+    "supported_languages": [
+       "pt"
+    ],
+    "social": {
+      "web": "https://skycraft.com.br",
```
Comment (DoJapHD) [inline minecraft_servers/skycraftbr/manifest.json:14]: Can't reach website

File: minecraft_servers/skycraftbr/manifest.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      "web_shop": "https://loja.skycraft.com.br",
```
Comment (DoJapHD) [inline minecraft_servers/skycraftbr/manifest.json:15]: Can't reach website

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/skycraftbr/background@2x.png
Comment (AimGear) [reply minecraft_servers/skycraftbr/background@2x.png:1]: Done

File: minecraft_servers/skycraftbr/manifest.json (hunk 2 above)
Comment (AimGear) [reply minecraft_servers/skycraftbr/manifest.json:15]: Both websites are offline temporarely, will be up soon.

Comment (DoJapHD) [reply minecraft_servers/skycraftbr/manifest.json:15]: How long is "soon"?

Comment (AimGear) [reply minecraft_servers/skycraftbr/manifest.json:15]: This week. We've rebuilt the server from scratch, so it'll take just a little time to set up the shop and site. It's a matter of days

Comment (DoJapHD) [reply minecraft_servers/skycraftbr/manifest.json:15]: I'll keep the PR open until then.
Just write a message in this PR once they're online

Comment (AimGear) [reply minecraft_servers/skycraftbr/manifest.json:15]: Isn't the website optional?

Comment (DoJapHD) [reply minecraft_servers/skycraftbr/manifest.json:15]: It is optional. Either you remove the keys until they're online or the PR stays open until they're online.

Comment (AimGear) [reply minecraft_servers/skycraftbr/manifest.json]: Done.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**